src/net/net.c

```diff
@@ -2359,15 +2359,15 @@ void NET_update_properties_step(Net* net, Branch* br, int t, Vec* var_values) {
 	net->bus_v_min[t] = v[k];
     }
 
-    // Voltage limit violations
-    //*************************
+    // Regulation voltage limit violations
+    //************************************
     dv = 0;
-    if (v[k] > BUS_get_v_max(bus))
-      dv = v[k]-BUS_get_v_max(bus);
-    if (v[k] < BUS_get_v_min(bus))
-      dv = BUS_get_v_min(bus)-v[k];
-    if (dv > net->bus_v_vio[t])
-      net->bus_v_vio[t] = dv;
+    if (v[k] > BUS_get_v_max_reg(bus))
+      dv = v[k]-BUS_get_v_max_reg(bus);
+    if (v[k] < BUS_get_v_min_reg(bus))
+      dv = BUS_get_v_min_reg(bus)-v[k];
+    if (dv > net->bus_v_reg_vio[t])
+      net->bus_v_reg_vio[t] = dv;
```

Comment:
Before maximum voltage limit violations were computed over all buses. Now maximum regulated voltage limit violations are computed but still over all buses. This needs to be change to be over all **regulated** buses.